packages/react/src/legend/LocationTotalsLegend.tsx

```diff
@@ -53,19 +56,19 @@ const LocationTotalsLegend = ({ diff, colors }: Props) => {
         <div style={styles.item.outer}>
           <Disc size={size} inner={pos.inner} outer={pos.inner} />
           {neg && <Disc size={size} inner={neg.inner} outer={neg.inner} />}
-          <div style={styles.item.caption}>outgoing ≅ incoming</div>
+          <div style={styles.item.caption}>{unequalText || 'outgoing ≅ incoming'}</div>
```

Comment:
We could then create a const `DEFAULT_TEXT_MESSAGES` with these values and then merge the prop into it: 
`const messages = { ...DEFAULT_TEXT_MESSAGES, ...textMessages }`